### PR TITLE
Created chat context menu

### PR DESCRIPTION
**Pull request by dmitry-zezik**

### CODE REVIEW

File: src/qml/pages/ChatPage.qml [hunk 1]
```diff
@@ -97,7 +97,31 @@ Page {
                     dropdown: true
                     currentIndex: -1
 
-                    Action {
+                    ContextMenuItem {
+                        text: qsTr("Call")
+                        iconName: "Make-Call"
+                        onTriggered: console.log("Chat deletion isn't implemented")
```
Comment (fpohtmeh-github) [inline src/qml/pages/ChatPage.qml:103]: copy+paste

File: src/qml/pages/ChatPage.qml [hunk 2]
```diff
@@ -97,7 +97,31 @@ Page {
...
+                    }
+
+                    ContextMenuItem {
+                        text: qsTr("Chat info")
+                        onTriggered: console.log("Chat deletion isn't implemented")
+                    }
+
+                    ContextMenuItem {
+                        text: qsTr("Mute notifications")
+                        onTriggered: console.log("Chat deletion isn't implemented")
```
Comment (fpohtmeh-github) [inline src/qml/pages/ChatPage.qml:116]: Hide all menu items that have no implementation, develop has working code only

File: src/qml/pages/ChatPage.qml (hunk 1 above)
Comment (dmitry-zezik) [reply src/qml/pages/ChatPage.qml:103]: Done

File: src/qml/pages/ChatPage.qml (hunk 2 above)
Comment (dmitry-zezik) [reply src/qml/pages/ChatPage.qml:116]: I have made them visible: false, but it doesn't work

Comment (fpohtmeh-github) [reply src/qml/pages/ChatPage.qml:116]: I will check this

Comment (fpohtmeh-github) [reply src/qml/pages/ChatPage.qml:116]: Item visibility works good in stdChatContextMenu. I don't see the problem @dmitry-zezik

**Review by fpohtmeh-github — APPROVED** [review]

**Pull request merged**